### PR TITLE
Chore: update general styles

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Extracts some general improvements from https://github.com/vtex-sites/base.store/pull/241
- Improves on `<Button variant="tertiary">` and aligns button icon
- Adds `title-small` class on `typography.scss`

|Before|After|
|-|-|
Before|After

### CODE REVIEW

**Review by lucasfp13 — APPROVED** [review]
LGTM!

File: src/styles/typography.scss [hunk 1]
```diff
@@ -84,6 +84,11 @@ h1, h2, h3, h4, h5, h6, p, ul {
   line-height: 1.25;
 }
 
+.tile-small {
```
Comment (filipewl) [inline src/styles/typography.scss:87]: Did you mean tile or **title**?

**Review by filipewl — APPROVED** [review]
LGTM but I think there's a typo!

File: src/styles/typography.scss (hunk 1 above)
Comment (hellofanny) [reply src/styles/typography.scss:87]: hhahaah yesh! Lemme fix it! Thanks!

**Pull request merged**